UI/source-tree.cpp

```diff
@@ -102,6 +102,7 @@ SourceTreeItem::SourceTreeItem(SourceTree *tree_, OBSSceneItem sceneitem_)
 	label->setSizePolicy(QSizePolicy::Expanding, QSizePolicy::Preferred);
 	label->setAlignment(Qt::AlignLeft | Qt::AlignVCenter);
 	label->setAttribute(Qt::WA_TranslucentBackground);
+	label->setEnabled(obs_sceneitem_visible(sceneitem));
```

Comment:
`obs_sceneitem_visible(sceneitem)` is called twice in the function, calling it once and re-using the result would be better.